docs/make.jl

```diff
@@ -15,21 +14,13 @@ const PAGES = [
             "users_guide/QuantumObject/QuantumObject.md",
             "users_guide/QuantumObject/QuantumObject_functions.md",
         ],
-        "Manipulating States and Operators" => [
-            "users_guide/states_and_operators/state_vectors.md",
-            "users_guide/states_and_operators/density_matrices.md",
-            "users_guide/states_and_operators/qubit_systems.md",
-            "users_guide/states_and_operators/expectation_values.md",
-            "users_guide/states_and_operators/superoperators.md",
-        ],
-        "Tensor Products and Partial Traces" => [
-            "users_guide/tensor_product/tensor.md",
-            "users_guide/tensor_product/partial_trace.md",
-        ],
+        "Manipulating States and Operators" => "users_guide/states_and_operators/states_and_operators.md",
```

Comment:
If there is only one file, maybe just put it in `users_guid/states_and_operators.md` (and remove the folder) ?